docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md

```diff
@@ -50,15 +75,15 @@ C:\>where dcap_quoteprov.dll
 C:\Windows\System32\dcap_quoteprov.dll
 ```
 
-## [Intel Data Center Attestation Primitives (DCAP) Libraries v1.9](https://registrationcenter-download.intel.com/akdlm/irc_nas/17362/Intel%20SGX%20DCAP%20for%20Windows%20v1.9.100.3.exe)
+## Intel Data Center Attestation Primitives (DCAP) Libraries
 
-After unpacking the self-extracting ZIP executable, you can refer to the *Intel SGX DCAP Windows SW Installation Guide.pdf*
-for more details on how to install the contents of the package.
+Windows Server 2019 should have this package installed by default via Windows Update.
+In that case, it is only necessary to set the registry key to allow the LC_driver to run. See step #1 in [Install the Intel DCAP driver](#install-the-intel-dcap-driver).
```

Comment:
What is "LC"?  Is this an acronym that can be expanded in text?

Reply:
This is the SGX Launch Configuration driver that is referenced in step #1, but I see where the confusion is coming from. I'll reword this sentence to clarify it. Thanks!